scripts/tests/qml_tests.sh

```diff
@@ -39,7 +71,20 @@ qmltest_compile || die
 print Y "Running the QML tests..."
 qmltest_run || die
 
+if [[ "$GRCOV_FILENAME" ]]; then
+  printn Y "Generating temp coverage file for lottie unit tests..."  
+  _grcov_qml qml_test_"$GRCOV_FILENAME"
+fi
+
 printn Y "Cleaning the existing project... "
 qmltest_cleanup || die
 
+if [[ "$GRCOV_FILENAME" ]]; then  
+  printn Y "merging temp files to qml_final_lcov.info... "
+  grcov -t lcov -o qml_final_lcov.info qml_test_"$GRCOV_FILENAME" || die "merging temp files to final failed"
```

Comment:
Well... in theory you should use `"$GRCOV_FILENAME"` instead of `qml_final_lcov.info`, right?